SQL/New_patches/2019-04-04_add_image_type_to_mri_protocol.sql

```diff
@@ -0,0 +1,8 @@
+-- Add an image_type column to be used for scan identification 
+-- in the mri_protocol and mri_protocol_violated_scans tables
+
+ALTER TABLE mri_protocol 
+  ADD COLUMN `image_type` varchar(255) default NULL;
```

Comment:
@cmadjar Is this image_type attributes to have limited number on possible values? 
Will all the possibles values be like the 2 examples you put in the descriptions?

Reply:
I don't like to add multiple varchar(255), specially when possible values range is limited. We might ran into problem due to extremely long row length.

I would have preferred a lookup table. Specially if the possible values are following a pattern with limited range.
I do understand that it might create more complication to let auto addition of possible values and/or forecast all possibilities.

I will thus approve anyway.